SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -176,6 +176,7 @@ const Schema_FieldId UNREAL_RPC_PAYLOAD_OFFSET_ID = 1;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_INDEX_ID = 2;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_PAYLOAD_ID = 3;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_TRACE_ID = 4;
+const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_UNIQUE_ID = 5;
```

Comment:
I don't see the rpc_payload schema updated in this PR - and please increment requireSetup